browser.js

```diff
@@ -193,9 +193,7 @@ define(function(require, exports, module) {
                 
                 var src = getIframeSrc(iframe);
                 var path = calcRootedPath(cleanIframeSrc(src));
-                
-                tab.title = 
-                tab.tooltip = "[B] " + path;
```

Comment:
Seemed like an unnecessary duplicate, unless I'm missing something? Do you see a scenario where tab's title needs to get overwritten on iframe loading?

PS, [not present](https://github.com/c9/c9.ide.preview.markdown/blob/master/markdown.js#L138-L145) in markdown!